context/app/api/vitessce_confs/base_confs.py

```diff
@@ -42,6 +45,8 @@ def get_conf_cells(self):
 
     def _replace_url_in_file(self, file):
         """Replace url in incoming file object
+        :param dict file: File dict which will have its rel_path replaced by url
+        rtype: dict The file with rel_path replaced by url
```

Comment:
```suggestion
        :rtype: dict The file with rel_path replaced by url
```
I'm not a master of pydoc, but change syntax, here and below.